rust/operator-binary/src/superset_controller.rs

```diff
@@ -265,13 +336,59 @@ fn build_node_rolegroup_service(
     })
 }
 
+/// The rolegroup [`ConfigMap`] configures the rolegroup based on the configuration given by the administrator
+fn build_rolegroup_config_map(
+    rolegroup: &RoleGroupRef<SupersetCluster>,
+    superset: &SupersetCluster,
+    rolegroup_config: &HashMap<PropertyNameKind, BTreeMap<String, String>>,
+    authentication_class: &Option<AuthenticationClass>,
+) -> Result<ConfigMap> {
+    let mut cm_conf_data = BTreeMap::new(); // filename -> filecontent
+
+    for property_name_kind in rolegroup_config.keys() {
```

Comment:
Is there any value in piping through the product-config machinery if we don't actually use it anyway?

Reply:
That's a good question which came also to my mind. Maybe I'm doing it wrong but i haven't understand the benefit of the product-config machinery yet.